### DISCUSSION TITLE
Which actor provided the voice of Gollum in the Lord of the Rings movie trilogy?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#620] [Galaxy Trivia] [answered]
*My precious*

**Comment by peterbaccaro** [comment]
English film actor, director and author **Andy Serkis** provided the voice of Gollum in the Lord of the Rings movie trilogy.

**Comment by thuongtruong1009** [comment, marked as answer]
The actor who provided the voice of Gollum in the Lord of the Rings movie trilogy is Andy Serkis. He also performed the motion capture for the character, using a combination of live-action performance and computer-generated imagery. Serkis's portrayal of Gollum was widely acclaimed, and he went on to reprise the role in the Hobbit film trilogy and to direct the film adaptation of The Jungle Book.
